plugins/de.ovgu.featureide.fm.attributes/src/de/ovgu/featureide/fm/attributes/base/impl/BooleanFeatureAttribute.java

```diff
@@ -81,4 +81,9 @@ public IFeatureAttribute cloneAtt(IFeature feature) {
 	public IFeatureAttribute cloneRecursive(IFeature feature) {
 		return new BooleanFeatureAttribute(feature, this.getName(), this.getUnit(), null, this.isRecursive(), this.isConfigurable());
 	}
+
+	@Override
+	public boolean isValidValue(String value) {
+		return true;
```

Comment:
is every string a valid value?

Reply:
This is intended, but open for discussion. Currently, anything resolves to false except capitalizations of true .